src/lib/hooks/useColor.ts

```diff
@@ -16,26 +16,23 @@ function UriForEthToken(address: string) {
  */
 async function getColorFromToken(token: Token, cb: (color: string | undefined) => void = () => void 0) {
```

Comment:
Changed this to accept `Currency` in #3081  but again - can probably address in that PR 

reasoning was basically mimic the existing useColor hook

Reply:
@ianlapham Did you run into #3085 in your build process when you did that?

Reply:
Yeah i think #3081 is the right time to do this - @JFrankfurt i actually didnt run into this somehow - but my build wasnt very stable can address in #3081